(BUG):Travis status tests is red

afaik the red is not caused directly by a change made duriing a PR but imho something has changed in the travis build env.

The build start failing once we commited a Readme.md modification ( so unrelated to tests).

```golang

=== RUN   TestAccLibvirtDomain_Basic

--- FAIL: TestAccLibvirtDomain_Basic (6.98s)

 libvirt_domain.acceptance-test-domain-1: Error creating libvirt domain: virError(Code=38, Domain=54, 

Message='Unable to write to '/sys/fs/cgroup/devices/machine/qemu-1-terraform-test.libvirt-qemu/devices.allow': Operation not permitted')

=== RUN   TestAccLibvirtDomain_Detailed
	* libvirt_domain.acceptance-test-domain-2: Error defining libvirt domain: virError(Code=9, Domain=20, Message='operation failed: domain 'terraform-test' already exists with uuid 025b377a-4770-4506-963a-ab66abb75bdf')
```

It seems that the plugin cannot cleanup resources after a testcase

afaik the red is not caused directly by a change made duriing a PR but imho something has changed in the travis build env.

The build start failing once we commited a Readme.md modification ( so unrelated to tests).

```golang

=== RUN   TestAccLibvirtDomain_Basic

--- FAIL: TestAccLibvirtDomain_Basic (6.98s)

	testing.go:459: Step 0 error: Error applying: 1 error occurred:

		

		* libvirt_domain.acceptance-test-domain-1: 1 error occurred:

		

		* libvirt_domain.acceptance-test-domain-1: Error creating libvirt domain: virError(Code=38, Domain=54, Message='Unable to write to '/sys/fs/cgroup/devices/machine/qemu-1-terraform-test.libvirt-qemu/devices.allow': Operation not permitted')

=== RUN   TestAccLibvirtDomain_Detailed

--- FAIL: TestAccLibvirtDomain_Detailed (6.58s)

	testing.go:459: Step 0 error: Error applying: 1 error occurred:

		

		* libvirt_domain.acceptance-test-domain-2: 1 error occurred:

		

		* libvirt_domain.acceptance-test-domain-2: Error defining libvirt domain: virError(Code=9, Domain=20, Message='operation failed: domain 'terraform-test' already exists with uuid 025b377a-4770-4506-963a-ab66abb75bdf')
```

--- a/travis/setup-host
+++ b/travis/setup-host
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -x
 
+sudo mknod /dev/kvm c 10 232
+
 sudo apt -t trusty-backports install -y lxd
 
 sudo sed -i 's/LXD_IPV4_ADDR=".*"/LXD_IPV4_ADDR="192.168.123.1"/' /etc/default/lxd-bridge
@@ -13,4 +15,5 @@ sudo service lxd restart
 
 sudo lxc launch ubuntu:zesty libvirt -c security.privileged=true
 sudo lxc config device add libvirt code disk source=`pwd` path=code
+sudo lxc config device add libvirt kvm unix-char path=/dev/kvm
 sudo lxc info libvirt

